lib/ansible/module_utils/network/ftd/fdm_swagger_client.py

```diff
@@ -506,7 +516,7 @@ def _is_enum(self, model):
         return self._is_string_type(model) and PropName.ENUM in model
 
     def _check_enum(self, status, model, value, path):
```

Comment:
can we change the name of `value` to `data` if feasible

Reply:
@justjais good catch. fixed.